org.fabric3.fabric.services.contribution.ContributionServiceMonitor#error=An error was encountered processing the contribution: {0}
org.fabric3.fabric.services.contribution.ContributionServiceMonitor#contributionWarnings=The following contribution warnings were reported: \n\n{0}
org.fabric3.fabric.services.contribution.ContributionTrackerMonitor#error=Error saving repository index: \n{0}
org.fabric3.fabric.services.contribution.ContributionReplayMonitor#error=Error restoring repository: \n{0}

